src/element-service.js

```diff
@@ -222,7 +233,7 @@ function waitForExtensionIfPresent(win, extension, head) {
 
   // TODO(jpettitt) investigate registerExtension to short circuit
   // the dom call in extensionScriptsInNode()
-  if (!extensionScriptsInNode(head).includes(extension)) {
+  if (!isExtensionScriptInNode(head)) {
```

Comment:
You should pass `extension` here right?

Reply:
I'd assume tests started to fail given this change. Might want to add some tests if not!

Reply:
right. gulp check-types task did catch this. Thanks all.